packages/shared/src/lib/dateFormat.ts

```diff
@@ -285,3 +286,27 @@ export const formatMonthYearOnly = (date: Date): string =>
     month: 'short',
     year: 'numeric',
   });
+
+export const formatDatesDuration = (start: Date, end: Date): string => {
+  const difference =
+    new Date(end || Date.now()).getTime() - new Date(start).getTime();
+  const differenceInMonths = Math.floor(
+    difference / (1000 * 60 * 60 * 24 * 30),
```

Comment:
Magic number

Reply:
Also don't we have this function already?
Maybe different format, but then naming is maybe wrong?

Or you can add this as type to the other one, so it's one generic maybe